doc/cephfs/fs-nfs-exports.rst

```diff
@@ -27,6 +27,15 @@ For more details on placement specification refer the `orchestrator doc
 <https://docs.ceph.com/docs/master/mgr/orchestrator/#placement-specification>`_.
 Currently only CephFS export type is supported.
 
+Update NFS Ganesha Cluster
+==========================
+
+.. code:: bash
+
+    $ ceph nfs cluster update <clusterid> <placement>
+
+This updates the deployed cluster according to the placement value.
```

Comment:
do we need this? As far as I can see, `ceph orch apply nfs <clusterid> --placement <placement>` should work as well, right? 

-1 for adding an alias here.

Reply:
Yes, the update takes care of pool and namespace. `orch apply nfs ...` requires pool and namespace right ?

Reply:
@varshar16 please give an example of `<placement>` and how that eventually gets applied through the orchestrator. I think that would help justify why we need this placement argument.

Reply:
Suppose we create cluster `test`:
`ceph nfs cluster create cephfs test`

Then update with:
cluster update: `ceph nfs cluster update test 3`

Values passed to orch
```(NFSServiceSpec({'placement': PlacementSpec(count=3), 'service_type': 'nfs', 'service_id': 'ganesha-test', 'unmanaged': False, 'pool': 'nfs-ganesha', 'namespace': 'test'})```

orch directly: `orch apply nfs test <pool> [<namespace>] [<placement>]`
User will have to specify pool and namespace